test/entrypoint.sh

```diff
@@ -24,3 +24,4 @@ curl -X 'GET' 'https://catalog.redhat.com/api/containers/v1/repositories/registr
 
 echo "Starting Integeration-Tests"
 bats $POLICY_PATH/conftest.sh
+exec $@
```

Comment:
Getting this error while running the docker run.
**/entrypoint.sh: line 27: exec: --: invalid option
exec: usage: exec [-cl] [-a name] [command [arguments ...]] [redirection ...]**

Reply:
yes, used the above docker file. `docker run --privileged --rm -it <docker image id> --env "SNYK_TOKEN=****" -v "path_app_source_code:/app:Z" hacbs-test`. I tried to run the scan using the docker build image id.

Reply:
what's  **<docker image id>* * in the cmd ? And it should be duplicated with "hacbs-test" at the end.

Reply:
pasted wrong cmd above.  here you go, `docker run --privileged --rm -it --env "SNYK_TOKEN=****" -v "path_app_source_code:/app:Z" <docker image id>` . could you also paste the output here.

Reply:
The output depends on the output of the snyk cmd.  If you didn't run it in a cluster with redhat-appstudio installed, you may need to pass the hosts file to the container by adding ' -v "/etc/hosts:/etc/hosts" to the above cmd. And below contents need to be included in the hosts file. 
23.194.131.216 registry.access.redhat.com
34.192.35.181 quay.io
23.52.40.67 catalog.redhat.com
35.190.124.19 deeproxy.snyk.io

Reply:
As Sri requested, can we have example output?

Reply:
With lines of  CMD ["snyk", "code test"] removed as suggested, the example output is as below. I am not sure if it is helpful. 
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 24089  100 24089    0     0  16188      0  0:00:01  0:00:01 --:--:-- 16177
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 34723  100 34723    0     0  26009      0  0:00:01  0:00:01 --:--:-- 26009
PASS
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  4284  100  4284    0     0   7175      0 --:--:-- --:--:-- --:--:--  7175
Starting Integeration-Tests
1..8
ok 1 /project/image/deprecated-labels
ok 2 /project/image/required-labels
ok 3 /project/image/optional-labels
ok 4 /project/image/inherited-labels_pass
ok 5 /project/image/inherited-labels_fail
ok 6 /project/clair/vulnerabilities-check
ok 7 /project/repository/deprecated-image
ok 8 /project/rpm_manifest/unsigned-rpms

Reply:
> I am not sure if it is helpful.
It sure is. Thanks!

Reply:
so we are not scanning hacbs-test repo then.